package.json

```diff
@@ -34,13 +35,14 @@
     "@react-native-community/async-storage": "^1.6.3",
     "@react-native-community/netinfo": "^3.2.1",
     "@sentry/react-native": "^1.0.9",
+    "@unimodules/core": "4.0.0",
```

Comment:
Likely won't matter in practice because they seem to mostly just bump the major version number, but let's copy the version range from `react-native-unimodules`: it's `~4.0.0`, which would include a 4.0.1 (but not a 4.1.0).